sql3/planner/planoptimizer.go

```diff
@@ -644,7 +644,13 @@ func tryToReplaceGroupByWithPQLAggregate(ctx context.Context, a *ExecutionPlanne
 					}
 					switch aggregable.AggExpression().Type().(type) {
 					case *parser.DataTypeID, *parser.DataTypeString:
-						return thisNode, true, nil
+						// if it is any other column other than _id bail
```

Comment:
Considering adding commentary on why the code was necessary, ie what behavior occurs if it wasnt there.